test/helpers/kubectl.go

```diff
@@ -1940,7 +1940,16 @@ func (kub *Kubectl) ciliumHealthPreFlightCheck() error {
 		return fmt.Errorf("cannot retrieve cilium pods: %s", err)
 	}
 	for _, pod := range ciliumPods {
-		status := kub.CiliumExec(pod, "cilium-health status -o json --probe")
+		var status *CmdRes
+
+		for i := 0; i < 10; i++ {
+			status = kub.CiliumExec(pod, "cilium-health status -o json --probe")
+			time.Sleep(10 * time.Second)
```

Comment:
I would move this sleep to after the break, and I'll do time.Sleep(i * time.Second)